torch_xla/experimental/pjrt.py

```diff
@@ -267,8 +267,11 @@ def broadcast_master_param(model: nn.Module) -> None:
   xm.mark_step()
 
 
-def rendezvous(tag: str, payload: bytes,
-               ordinals: Optional[List[int]]) -> List[bytes]:
+def rendezvous(tag: str,
+               payload: bytes,
+               ordinals: Optional[List[int]],
+               *,
```

Comment:
python noob, what does this * mean?

Reply:
This makes `mesh_port` a [keyword-only argument](https://peps.python.org/pep-3102/)